temporal/api/cloud/cloudservice/v1/request_response.proto

```diff
@@ -619,3 +622,33 @@ message DeleteServiceAccountResponse {
     // The async operation.
     temporal.api.cloud.operation.v1.AsyncOperation async_operation = 1;
 }
+
+message GetUsageRequest {
+    // Filter for UTC time >= - optional.
+    // Defaults to: start of the current month.
+    // Must be: within the last 90 days from the current date.
+    // Must be: midnight UTC time.
```

Comment:
Meaning, if any timestamp that is not midnight UTC time--regardless of the date--validation will fail?

Reply:
That is correct, currently it's a strict requirement since we only support `Day` granularity